### ISSUE TITLE
upload file error

### ISSUE THREAD

**Issue by hrxiang** [assemmarwan/dio_http_formatter#3]
```
E/flutter ( 9602): [ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: NoSuchMethodError: Class 'FormData' has no instance getter 'isNotEmpty'.
E/flutter ( 9602): Receiver: Instance of 'FormData'
E/flutter ( 9602): Tried calling: isNotEmpty
E/flutter ( 9602): #0      Object.noSuchMethod (dart:core-patch/object_patch.dart:63:5)
E/flutter ( 9602): #1      HttpFormatter._prepareLog (package:dio_http_formatter/src/dio_http_formatter_base.dart:111:32)
E/flutter ( 9602): #2      HttpFormatter.onResponse (package:dio_http_formatter/src/dio_http_formatter_base.dart:57:23)
```

**Comment by assemmarwan** [comment]
@hrxiang I have deployed a fix (2.0.1). Please verify if the issue is resolved 😄

**Comment by assemmarwan** [comment]
Closing this for now
